tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = py311
+
+[testenv]
+extras = dev
+commands = pytest -s tests
```

Comment:
I think we should be able to move this config into `pyproject.toml` as well according to the [tox docs](https://tox.wiki/en/4.11.4/config.html#pyproject-toml).